photo/tests/test_views.py

```diff
@@ -121,7 +125,8 @@ class TestSubmissionsFromContestDelete(TestCase):
     
     def setUp(self):
         self.user = UserFactory()
-        self.contest = ContestFactory()
+        self.creator = UserFactory()
```

Comment:
In this test cases I think there is no need to create the "creator", we can use the user in the ContestFactory param, right?

Reply:
I added a creator because then the submission is done by a different user. It seems a more accurate use of those actions.